### PR TITLE
Properly initialize OneToN address if not given by user

### LINKED ISSUE #4187: Scenario PFS7 fails
Tracking issue to get the PFS7 scenario running.

### PR DESCRIPTION
**Pull request by palango**
I think this should fix #4187

### CODE REVIEW

File: raiden/ui/app.py [hunk 1]
```diff
@@ -236,7 +240,8 @@ def run_app(
             config=config,
             chain=blockchain_service,
             query_start_block=BlockNumber(start_block),
-            default_one_to_n_address=one_to_n_contract_address,
+            default_one_to_n_address=one_to_n_contract_address
+            or contracts[CONTRACT_ONE_TO_N]["address"],
```
Comment (ulope) [inline raiden/ui/app.py]: This looks very weird, maybe adding parens would make it more readable (or does black remove those?)

File: raiden/tests/integration/cli/conftest.py [hunk 2]
```diff
@@ -107,6 +107,8 @@ def cli_args(request, tmpdir, raiden_testchain, removed_args, changed_args, envi
             initial_args["service_registry_contract_address"],
             "--environment-type",
             environment_type,
+            "--one-to-n-contract-address",
```
Comment (ulope) [inline raiden/tests/integration/cli/conftest.py]: Is this necessary? The loop below should do that anyway if it's in `initial_args`

Comment (palango) [reply raiden/tests/integration/cli/conftest.py]: Good point, removed the unnecessary ones.

File: raiden/ui/app.py (hunk 1 above)
Comment (palango) [reply raiden/ui/app.py]: I copied the style from below, but added parens now.

**Pull request merged**